roles/kubernetes-apps/external_provisioner/local_volume_provisioner/tasks/main.yml

```diff
@@ -2,13 +2,15 @@
 
 - name: Local Volume Provisioner | Ensure base dir is created on all hosts
```

Comment:
Is it necessary to have this task here as the creation of the directory is already handled in preinstall?

Reply:
I left that in because I assume the original author wanted the role to work standalone even when `preinstall` isn't ran. It should be a no-op as long as preinstall created the directories consistently already.